src/chrome-launcher.ts

```diff
@@ -56,7 +56,10 @@ export interface ModuleOverrides {
 
 const sigintListener = async () => {
   for (const instance of instances) {
-    await instance.kill();
+    try {
+      await instance.kill();
+    } catch (err) {
```

Comment:
A log here I think would be a good idea instead of silently eating the error.

Reply:
The only reject path is already logging the error, would you prefer it moved up here?